G4EMPH/g4gen_job.fcl

```diff
@@ -24,7 +24,7 @@ source:
 {
   module_type: EmptyEvent
   maxEvents: 10000
-  firstRun:  1293
+  firstRun:  2293
```

Comment:
Did you intend to change this run number? Will it break things for anyone else using this fcl?

Reply:
This shouldn't break anything, but might surprise people who assume they are generating a Phase 1b run.